packages/teams-js/test/private/files.spec.ts

```diff
@@ -398,21 +399,23 @@ describe('files', () => {
       await utils.initializeWithContext('content');
       utils.setRuntimeConfig({ apiVersion: 1, supports: {} });
 
-      expect(() =>
+      try {
         files.openCloudStorageFile(
           mockCloudStorageFolderItem,
           files.CloudStorageProvider.Box,
           FileOpenPreference.Inline,
-        ),
-      ).toThrowError(errorNotSupportedOnPlatform);
+        );
+      } catch (e) {
+        expect(e).toEqual(errorNotSupportedOnPlatform);
+      }
     });
   });
   describe('openFilePreview', () => {
     it('openFilePreview should throw error when files capability is not supported', async () => {
       await utils.initializeWithContext('content');
```

Comment:
Please add `expect.assertions(1)` to validate `expect` in catch block is called.